src/strings/en-US/PowerFxResources.resw

```diff
@@ -2354,11 +2354,23 @@
     <value>index</value>
     <comment>function_parameter - Second argument of the Index function - Number type.</comment>
   </data>
+  <data name="AboutIndexT" xml:space="preserve">
```

Comment:
you can reuse the arg and about strings that Bryan added for Index

Reply:
done